lib/imageflow/graph_runner.ex

```diff
@@ -29,10 +30,13 @@ defmodule Imageflow.GraphRunner do
   defp add_outputs(job, inputs) do
     inputs
     |> Enum.reduce_while(:ok, fn
-      {id, _}, :ok ->
-        with :ok <- Native.add_output_buffer(job, id) do
-          {:cont, :ok}
-        else
+      {id, value}, :ok ->
+        case value do
+          {:file, _path} -> Native.add_output_buffer(job, id)
+          :bytes -> Native.add_output_buffer(job, id)
```

Comment:
I'm not understanding what the need for this `case` is. In both scenarios, you're doing the exact same thing. Why isn't the existing `with` enough?

Reply:
My bad, I was trying something and forgot to switch it back.